### ISSUE TITLE
Execute `backup full` error `exit status 1`

### ISSUE THREAD

**Issue by foesa-yang** [vesoft-inc/nebula-br#12]
![截屏2021-07-22 上午11 42 36](https://user-images.githubusercontent.com/35463247/126587391-48e16d8e-aa9e-46d3-9b76-3bae0be5b978.png)
Deployed by docker-compose. The following is the meta service information.
![截屏2021-07-22 上午11 43 24](https://user-images.githubusercontent.com/35463247/126587404-b388356e-3004-4f16-b91d-a39209f2398b.png)

**Comment by linkensphere201** [comment]
good catch, `br` is not recommended to used in docker environment for now. I'll test it later. @foesa-yang
